.github/workflows/ci-app-examples.yml

```diff
@@ -12,7 +12,8 @@ on:
       - "src/lightning_app/**"
       - "tests/tests_app_examples/**"
       - "examples/app_*"
-      - "requirements/app/**"
+      - "requirements/app/*"
```

Comment:
Why remove the **?

Reply:
I think that the depth is just 1 not any